lego/apps/emojis/views.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import mixins, viewsets
+
+from lego.apps.emojis.models import Emoji
+from lego.apps.emojis.permissions import EmojiPermissionHandler
+from lego.apps.emojis.serializers import EmojiSerializer
+from lego.apps.permissions.api.views import AllowedPermissionsMixin
+
+
+class EmojiViewSet(AllowedPermissionsMixin, viewsets.ModelViewSet):
+    pagination_class = None
```

Comment:
Is there supposed to be pagination here?
From PR description:
> -  HTTP GET /api/v1/emojis to list all of the Emojis (paginated).

Reply:
Brain broke, I mean to write that there's no pagination.